### PR TITLE
Support CompiledModel.outputs property

### PR DESCRIPTION
**Pull request by almilosz**
### Details:
 - Enable compileModel.outputs for 001-hello-world notebook and update the notebook
 - Create binding for `Output<const Node>` class and expose `get_any_name` method
 - Expose `get_tensor(Output<const Node>` method of the InferenceRequest class 

### Tickets:
 - *113696*

### CODE REVIEW

File: src/bindings/js/node/src/compiled_model.cpp [hunk 1]
```diff
@@ -37,3 +42,16 @@ Napi::Value CompiledModelWrap::create_infer_request(const Napi::CallbackInfo& in
     ov::InferRequest infer_request = _compiled_model.create_infer_request();
     return InferRequestWrap::Wrap(info.Env(), infer_request);
 }
+
+Napi::Value CompiledModelWrap::get_outputs(const Napi::CallbackInfo& info) {
+    auto cm = CompiledModelWrap::get_compiled_model();
+    auto cm_outputs = cm.outputs();  // Output<Node>
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/src/compiled_model.cpp:48]:
```suggestion
    auto cm_outputs = _compiled_model.outputs();  // Output<Node>
```

File: src/bindings/js/node/include/compiled_model.hpp [hunk 2]
```diff
@@ -31,6 +31,7 @@ class CompiledModelWrap : public Napi::ObjectWrap<CompiledModelWrap> {
     static Napi::Object Init(Napi::Env env, Napi::Object exports);
 
     void set_compiled_model(const ov::CompiledModel& compiled_model);
+    ov::CompiledModel get_compiled_model() const;
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/include/compiled_model.hpp:34]: Why this method needed?

File: src/bindings/js/node/src/node_output.cpp [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#include "node_output.hpp"
+
+Output::Output(const Napi::CallbackInfo& info) : Napi::ObjectWrap<Output>(info) {}
+
+Napi::Function Output::GetClassConstructor(Napi::Env env) {
+    return DefineClass(
+        env,
+        "Output",
+        {InstanceMethod("getAnyName", &Output::get_any_name), InstanceMethod("toString", &Output::get_any_name)});
```
Comment (vishniakov-nikolai) [inline src/bindings/js/node/src/node_output.cpp:9]: 👍

**Review by vishniakov-nikolai — CHANGES_REQUESTED** [review]
Minor comment. I think you can remove this methods: 
```cpp
    void set_compiled_model(const ov::CompiledModel& compiled_model);
    ov::CompiledModel get_compiled_model() const;
``` 
by using **_compiled_model** directly.

File: src/bindings/js/node/src/compiled_model.cpp (hunk 1 above)
Comment (almilosz) [reply src/bindings/js/node/src/compiled_model.cpp]: Getter here is unnecessary

**Pull request merged**